ds-controller-board/res/configuration.toml

```diff
@@ -85,4 +78,12 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
   Retained = "false"
   AutoReconnect = "true"
   ConnectTimeout = "5" # Seconds
-  SkipCertVerify = "false"
\ No newline at end of file
+  SkipCertVerify = "false"
+
+[AppCustom]
+  [DriverConfig]
```

Comment:
Why need for extra level. I would just have the settings under `[AppCustom]` or rename `[AppCustom]` to `[DriverConfig]`. The name AppCustom originated for an example, but can be anything...